syft/generic/frameworks/hook/hook.py

```diff
@@ -471,9 +471,9 @@ def overloaded_native_method(self, *args, **kwargs):
 
                 # if object is a pointer of pointer, set register to False
                 if isinstance(self.child, PointerTensor):
-                    wrap_args = {"register": False}
+                    wrap_args = {"register": False, **self.get_class_attributes()}
```

Comment:
What's this register? I don't see it in class constructors...
Also, I saw it in args to `create_pointer` methods but they aren't used in these methods at all

Reply:
Don't know what it is, didn't had time to investigate. Do you want me to remove it and see?

Reply:
Haha, we can try! If I didn't miss anything, it shouldn't break 🤞